pages/nft-book-store/send/[classId].vue

```diff
@@ -199,9 +208,17 @@ const orderInfo = ref<any>({})
 const nftImage = ref('')
 
 const userIsOwner = computed(() => wallet.value && ownerWallet.value === wallet.value)
-const isSendButtonDisabled = computed(() => isEditingNFTId.value || isLoading.value || isVerifyingNFTId.value || isAutoFetchingNFTId.value || !!nftIdError.value)
+const isSendButtonDisabled = computed(() => isEditingNFTId.value || isLoading.value || isVerifyingNFTId.value || isAutoFetchingNFTId.value || !!nftIdError.value || isLimitReached.value)
 
 const nftClassName = computed(() => nftStore.getClassMetadataById(classId.value as string)?.name)
+const messageCharCount = computed(() => {
+  let count = 0
+  for (let i = 0; i < memo.value.length; i++) {
+    count += isFullWidth(memo.value[i]) ? 2 : 1
+  }
+  return count
+})
+const isLimitReached = computed(() => { return messageCharCount.value > AUTHOR_MESSAGE_LIMIT })
```

Comment:
I think these 2 functions can be a [composable](https://vuejs.org/guide/reusability/composables#composables)

Reply:
[♻️ Refactor char count computed to composables](https://github.com/likecoin/nft-book-press/pull/212/commits/07a063f483d706b134ffd27e42dfe599c3b335b4)